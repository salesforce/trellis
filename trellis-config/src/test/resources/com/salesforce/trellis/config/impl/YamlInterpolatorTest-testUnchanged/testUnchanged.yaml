whitelists:
- file: foo/bar/baz.yaml
  headerComment: foo bar baz
  action: ALLOW

groups:

  - name: PE_IMPLS
    includes:
      - sfdc.shared:baz
      - sfdc.core:platform-encryption
      - sfdc.shared:bar
    except:
      - sfdc.ui:*
      - sfdc.shared:foo

  - name: PE_APIS
    includes:
      - sfdc.core:platform-encryption-api

rules:

  - action: DENY
    from:
      - myapp:from
    exceptFrom:
      - myapp:exceptFrom
    to:
      - myapp:to
    exceptTo:
      - myapp:exceptTo
    reason: hello!
    distance: TRANSITIVE_ONLY
    optionality: OPTIONAL_ONLY
    scope: compile
