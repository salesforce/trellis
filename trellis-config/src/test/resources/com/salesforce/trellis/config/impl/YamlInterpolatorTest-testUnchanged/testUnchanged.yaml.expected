whitelists:

- file: foo/bar/baz.yaml
  action: ALLOW
  headerComment: foo bar baz

groups:

- name: PE_IMPLS
  includes:
  - sfdc.shared:baz
  - sfdc.core:platform-encryption
  - sfdc.shared:bar
  except:
  - sfdc.ui:*
  - sfdc.shared:foo

- name: PE_APIS
  includes:
  - sfdc.core:platform-encryption-api

rules:

- action: DENY
  from:
  - myapp:from
  exceptFrom:
  - myapp:exceptFrom
  to:
  - myapp:to
  exceptTo:
  - myapp:exceptTo
  reason: hello!
  distance: TRANSITIVE_ONLY
  scope: compile
  optionality: OPTIONAL_ONLY
